assets/shell-integration/wezterm.sh

```diff
@@ -380,7 +383,11 @@ fi;
 
 } # end of __wezterm_install_bash_prexec
 
-__wezterm_install_bash_prexec
+# blesh provides it's own preexec mechanism which is recommended over bash-preexec
+# See https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A71-Introduction#user-content-fn-blehook for more details
+if [[ ! -v BLE_VERSION ]]; then
```

Comment:
`[[ -v var ]]` is Bash 4.2 feature. Unfortunately, this will produce error messages in lower versions of Bash such as Bash 3.2 in macOS. `[[ ! ${BLE_VERSION-} ]]` is the way that works in all the versions of Bash 3.0+.

Reply:
I pushed https://github.com/wez/wezterm/commit/78ea214a96da7e9e8c8e5f3539bb788902eaff53 which uses the same `-n "$foo"` test that we use for testing for zsh

Reply:
Thanks. I usually don't give a damn about the ancient versions of Darwin.